raft.c

```diff
@@ -260,13 +259,15 @@ static int raftSendRequestVote(raft_server_t *raft, void *user_data,
 
     /* RAFT.REQUESTVOTE <src_node_id> <term> <candidate_id> <last_log_idx> <last_log_term> */
     if (redisAsyncCommand(ConnGetRedisCtx(node->conn), handleRequestVoteResponse,
-                node, "RAFT.REQUESTVOTE %d %d %d:%d:%d:%d",
+                node, "RAFT.REQUESTVOTE %d %d %d:%d:%d:%d:%d",
                 raft_node_get_id(raft_node),
                 raft_get_nodeid(raft),
                 msg->term,
                 msg->candidate_id,
                 msg->last_log_idx,
-                msg->last_log_term) != REDIS_OK) {
+                msg->last_log_term,
+                raft_is_timeout_now(raft)
```

Comment:
I think you may set msg->transfer_leader in the library so lib user wouldn't need to call this function. It'll just use it as another variable here as msg->transfer_leader.